landing desktop header

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<html>
+<body>
+    <h1 style="background-color:rgb(30, 124, 109);color:white;">My Tenent</h1>
```

Comment:
Don't use inline styles

Reply:
Dont commit or create pill request from github.com

Use GitHub desktop to push the changes and the go to github.com and the create a new pull request